public/app/features/alerting/unified/RuleEditorGrafanaRules.test.tsx

```diff
@@ -61,9 +64,13 @@ const mocks = {
   },
 };
 
+const server = setupServer();
```

Comment:
Can we replace some of this logic with `setupMswServer()` which will automatically set up the jest lifecycle hooks too? 

https://github.com/grafana/grafana/blob/3cfe5b5bea0e71296afa07a00e2bde8b4c2a8c82/public/app/features/alerting/unified/mockApi.ts#L412-L430

Reply:
Good idea, thanks :+1: I've replaced the uses of it within the alerting code